### PR TITLE
chore: update multisig addresses

### PR DESCRIPTION
**Pull request by howydev**
# Pull Request Checklist

- [ ] Did you add new tests and confirm existing tests pass? (`yarn test`)
- [ ] Did you update relevant docs? (docs are found in the `site` folder, and guidelines for updating/adding docs can be found in the [contribution guide](https://github.com/alchemyplatform/aa-sdk/blob/main/CONTRIBUTING.md))
- [ ] Do your commits follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) standard?
- [ ] Does your PR title also follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) standard?
- [ ] If you have a breaking change, is it [correctly reflected in your commit message](https://www.conventionalcommits.org/en/v1.0.0/#examples)? (e.g. `feat!: breaking change`)
- [ ] Did you run lint (`yarn lint:check`) and fix any issues? (`yarn lint:write`)
- [ ] Did you follow the [contribution guidelines](https://github.com/alchemyplatform/aa-sdk/blob/main/CONTRIBUTING.md)?

---

## PR-Codex overview
The focus of this PR is to update addresses in the `utils.ts` and `config.ts` files related to various chains and plugins.

### Detailed summary
- Updated addresses for various chains in `utils.ts`
- Added new addresses for different chains in `config.ts`
- Imported chain identifiers in `config.ts`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Review by rthomare — APPROVED** [review]

File: packages/accounts/src/msca/plugins/multisig/plugin.ts [hunk 1]
```diff
@@ -154,7 +154,16 @@ export type MultisigPluginActions<
   ReadAndEncodeActions<TAccount>;
 
 const addresses = {
-  11155111: "0x0b135A12EB2f7b441DCcE5F7DE07DB65AE7c4649" as Address,
+  1: "0x000000000000A53f64b7bcf4Cd59624943C43Fc7" as Address,
```
Comment (rthomare) [inline packages/accounts/src/msca/plugins/multisig/plugin.ts:157]: nit you can use [<chain>.id] here.

Comment (howydev) [reply packages/accounts/src/msca/plugins/multisig/plugin.ts:157]: TIL. this is autogenerated by plugingen though, probably best to leave it as is

Comment (howydev) [reply packages/accounts/src/msca/plugins/multisig/plugin.ts:157]: could i do that change above in config.ts?

Comment (denniswon) [reply packages/accounts/src/msca/plugins/multisig/plugin.ts:157]: no this is fine. auto-geneated file.

**Comment by denniswon** [comment]
LGTM @adam-alchemy thanks

**Comment by howydev** [comment]
@denniswon prefer to merge after e2e passes just in case the last min changes to the contract between the 2 deployments affects some of the tooling here.. else if theres an issue it would be caught by one of our partners while integrating msig

**Comment by adam-alchemy** [comment]
merging for now because it's broken anyways on main, can fix in a subsequent PR

**Pull request merged**